packages/ui/src/ui/pages/operations/OperationDetail/tabs/Jobs/OperationJobsTable/OperationJobsTable.scss

```diff
@@ -16,7 +16,6 @@
                 &_type {
```

Comment:
probably should be camel case
https://nda.ya.ru/t/_Q7yasgJ6xbXD9

Reply:
At the moment we use `hammer.format.ReadableField`, it is common approach to format string values. And I think it is better to use it here too.